test/test_transforms.py

```diff
@@ -86,7 +86,7 @@ def test_scriptmodule_AmplitudeToDB(self):
         _test_script_module(transforms.AmplitudeToDB, spec)
 
     def test_scriptmodule_MelScale(self):
-        spec_f = torch.rand((1, 6, 201))
+        spec_f = torch.rand((1, 201, 6))
```

Comment:
We don't need to change this. The important part is just that it tests with a random matrix.